mLRS/Common/setup.h

```diff
@@ -396,6 +411,20 @@ void setup_configure(void)
     case SETUP_FREQUENCY_BAND_868_MHZ:
         Config.FhssNum = FHSS_NUM_BAND_868_MHZ;
         break;
+    case SETUP_FREQUENCY_BAND_433_MHZ:
+        switch (Config.Mode) {
+        case MODE_31HZ: Config.FhssNum = FHSS_NUM_BAND_433_MHZ; break;
+        case MODE_19HZ: Config.FhssNum = FHSS_NUM_BAND_433_MHZ_19HZ_MODE; break;
+        default:
+            while (1) {} // must not happen, should have been resolved in setup_sanitize()
+        }
+    case SETUP_FREQUENCY_BAND_70_CM_HAM:
```

Comment:
BUG BUG BUG
`break;` is missing here !!!!

Reply:
Fixed